modules/transactions/src/integrationTest/java/org/apache/ignite/internal/tx/readonly/ItReadOnlyTxInPastTest.java

```diff
@@ -80,4 +111,38 @@ void implicitReadOnlyTxDoesNotLookBeforeTableCreation() {
 
         assertThat(count, is(0L));
     }
+
+    private void setDefaultZoneAutoAdjustScaleUpTimeoutToImmediate() {
```

Comment:
Minor: Duplicating code in tests is generally OK. However, we may move this code to a base class (`ClusterPerClassIntegrationTest`, for instance) and execute it right after cluster initialization when the colocation feature is enabled. WDYT?

Reply:
I've considered such option, however `ClusterPerClassIntegrationTest` doesn't seem as a proper place, meaning that "setDefaultAutoAdjustScaleUpTimeoutToImmediate" is a bit specific to be placed in general ClusterPerClassIntegrationTest. Not sure though.

Reply:
:ok_hand: